doc/en/components/scheduling/date-range-picker.md

```diff
@@ -0,0 +1,418 @@
+```
+
+After that, you need to import the `DateRangePicker`, its necessary CSS, and register its module, as follows:
+
+```ts
+import { defineComponents, IgcDateRangePickerComponent } from 'igniteui-webcomponents';
+import 'igniteui-webcomponents/themes/light/bootstrap.css';
+
+defineComponents(IgcDateRangePickerComponent);
+```
+
+<!-- end: WebComponents -->
+
+Now you can start with a basic configuration of the {Platform} `DateRangePicker`.
+
+```html
+<igc-date-range-picker anchor="hover-button">
+  Congrats you've hovered the button!
+</igc-date-range-picker>
+
+<igc-button id="hover-button">Hover me</igc-button>
+```
+
+For a complete introduction to the {ProductName}, read the [*Getting Started*](../general-getting-started.md) topic.
+
+## Usage
+
+The `DateRangePicker` allows users to select a start and end date either by choosing a date range from a dropdown/calendar pop-up or by typing directly into the input fields - one for the start date and one for the end date.
+
+The picker offers two modes for displaying date values: single input and two inputs. In single input mode, the field is read-only, and the date range cannot be edited by typing. In two inputs mode, however, users can edit the start and end dates by typing in separate input fields.
+
+When the calendar is visible, a date range can be selected by choosing both a start and end date. Selecting a date will set the start date, and once a second date is chosen, it will set the end date. If a range is already selected, clicking any other date on the calendar will start a new range selection.
+
+### Display Date Range Picker
+
+To instantiate a `DateRangePicker` in its default single input mode, use the following code:
+
+```html
+<igc-date-range-picker>
+</igc-date-range-picker>
+```
+
+To switch the `DateRangePicker` to use two inputs, set the `UseTwoInputs` property to `true`.
+
+```html
+<igc-date-range-picker use-two-inputs="true">
+</igc-date-range-picker>
+```
+
+### Value
+
+In addition to being selected or typed by the user, the range value of the `DateRangePicker` can also be set using the `Value` property. It's important to note that the value must follow the format: **{ start: startDate, end: endDate }**, where `startDate` and `endDate` are `Date` objects representing the selected range.
+
+```ts
+let dateRange = document.querySelector('igc-date-range-picker') as IgcDateRangePickerComponent;
+let startDate: Date = new Date(2025, 4, 6);
+let endDate: Date = new Date(2025, 4, 8);
+dateRange.value = { start: startDate, end: endDate }
+```
+
+### Read-only
+
+You can also make the `DateRangePicker` read-only, which disables changing the range value through both typing and calendar selection. This is useful when the range is assigned via the value attribute and is intended to be display-only. To enable this behavior, simply set the `ReadOnly` property.
+
+```html
+<igc-date-range-picker use-two-inputs="true" readonly>
+</igc-date-range-picker>
+```
+
```

Comment:
[These details were elaborated](https://github.com/IgniteUI/igniteui-webcomponents/wiki/Date-Range-Picker#8-readonly-state) later in the process of design&implementation and can be added here.
Also we can mention the `nonEditable` property, which as opposed to `readOnly`, only makes the input non-editable by typing, while still allowing selection through the calendar & clearing trough the clear icon.